### ISSUE TITLE
Baileys not working

### ISSUE THREAD

**Issue by ThizPersonDoesNotExist** [TOXIC-DEVIL/WhatsAsenaPublic#6]
baileys getting error : 

>  ℹ️ Connecting to WhatsApp... Please Wait.
>  WARN [2021-06-15 19:06:33.838 +0000] (103): connect attempt 1 failed: Error: close, retrying...
> class: "Baileys"
> error: {}
> WARN [2021-06-15 19:06:37.981 +0000] (103): connect attempt 2 failed: Error: close, retrying...
> class: "Baileys"
> error: {}
> WARN [2021-06-15 19:06:42.102 +0000] (103): connect attempt 3 failed: Error: close, retrying...
> class: "Baileys"
> error: {}
> WARN [2021-06-15 19:06:46.328 +0000] (103): connect attempt 4 failed: Error: close, retrying...
> class: "Baileys"
> error: {}
> WARN [2021-06-15 19:06:50.396 +0000] (103): connect attempt 5 failed: Error: close, retrying...
> class: "Baileys"
> error: {}
> WARN [2021-06-15 19:06:54.539 +0000] (103): connect attempt 6 failed: Error: close, retrying...
> class: "Baileys"
> error: {}
> WARN [2021-06-15 19:06:58.704 +0000] (103): connect attempt 7 failed: Error: close, retrying...
> class: "Baileys"
> error: {}
> WARN [2021-06-15 19:07:02.831 +0000] (103): connect attempt 8 failed: Error: close, retrying...
> class: "Baileys"
> error: {}
> WARN [2021-06-15 19:07:06.984 +0000] (103): connect attempt 9 failed: Error: close, retrying...
> class: "Baileys"
> error: {}
> WARN [2021-06-15 19:07:11.206 +0000] (103): connect attempt 10 failed: Error: close
> class: "Baileys"
> error: {}
> 2021-06-15T19:07:11.217455+00:00 app[worker.1]: Tue Jun 15 2021 22:07:11 GMT+0300 (GMT+03:00)
> 2021-06-15T19:07:11.217568+00:00 app[worker.1]: Program node bot.js exited with code 0 

Tried other Asena Sessions,
created new app on heroku all over again,
but it keep getting this error ? what should i do?

**Comment by ThizPersonDoesNotExist** [comment]
Never mind.  I fixed it. 
There was some issue with baileys and some modification in repo was also required.

**Comment by TOXIC-DEVIL** [comment]
> baileys getting error :
> 
> > ℹ️ Connecting to WhatsApp... Please Wait.
> > WARN [2021-06-15 19:06:33.838 +0000] (103): connect attempt 1 failed: Error: close, retrying...
> > class: "Baileys"
> > error: {}
> > WARN [2021-06-15 19:06:37.981 +0000] (103): connect attempt 2 failed: Error: close, retrying...
> > class: "Baileys"
> > error: {}
> > WARN [2021-06-15 19:06:42.102 +0000] (103): connect attempt 3 failed: Error: close, retrying...
> > class: "Baileys"
> > error: {}
> > WARN [2021-06-15 19:06:46.328 +0000] (103): connect attempt 4 failed: Error: close, retrying...
> > class: "Baileys"
> > error: {}
> > WARN [2021-06-15 19:06:50.396 +0000] (103): connect attempt 5 failed: Error: close, retrying...
> > class: "Baileys"
> > error: {}
> > WARN [2021-06-15 19:06:54.539 +0000] (103): connect attempt 6 failed: Error: close, retrying...
> > class: "Baileys"
> > error: {}
> > WARN [2021-06-15 19:06:58.704 +0000] (103): connect attempt 7 failed: Error: close, retrying...
> > class: "Baileys"
> > error: {}
> > WARN [2021-06-15 19:07:02.831 +0000] (103): connect attempt 8 failed: Error: close, retrying...
> > class: "Baileys"
> > error: {}
> > WARN [2021-06-15 19:07:06.984 +0000] (103): connect attempt 9 failed: Error: close, retrying...
> > class: "Baileys"
> > error: {}
> > WARN [2021-06-15 19:07:11.206 +0000] (103): connect attempt 10 failed: Error: close
> > class: "Baileys"
> > error: {}
> > 2021-06-15T19:07:11.217455+00:00 app[worker.1]: Tue Jun 15 2021 22:07:11 GMT+0300 (GMT+03:00)
> > 2021-06-15T19:07:11.217568+00:00 app[worker.1]: Program node bot.js exited with code 0
> 
> Tried other Asena Sessions,
> created new app on heroku all over again,
> but it keep getting this error ? what should i do?

Yes, We know that it was an issue from baileys and it have been fixed on this ropo & Thank You For Reporting Us ❤
